test/runtests.jl

```diff
@@ -87,7 +87,7 @@ end
     end
 
     for T in (Int, UInt, Int8, UInt16, Float32, Float64), _ in 1:5
-        x = round(T, 100*rand())
+        x = T(round(100*rand()))
```

Comment:
```suggestion
        x = T(rand(0:100))
```
Looking at it afresh, I've realized that one doesn't need `round` at all.